docs/visualization/plotting-1d-data.ipynb

```diff
@@ -110,7 +110,9 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "We can always plot just a single item of the dataset:"
+    "We can always plot just a single item of the dataset.\n",
+    "There is nothing special that needs to be done for plot itself to allow this.\n",
```

Comment:
I'm not sure how much information these 2 extra sentences add. I think the code example is rather self-explanatory?

Reply:
I was trying to distinguish between plot options, and things that are core to scipp. I don't have a particularly strong preference, so will remove